src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -263,7 +263,20 @@ void java_bytecode_convert_method_lazy(
   symbol_tablet &symbol_table)
 {
   symbolt method_symbol;
-  typet member_type=java_type_from_string(m.descriptor);
+  typet member_type;
```

Comment:
This is of course shouldn't go in as part of this PR and instead we should wait for TG-865 before merging this PR.

Reply:
Indeed, that's why I left this as the last patch in the series, so that it can easily be dropped before merging.